ecodse_funtime_alpha/train.py

```diff
@@ -16,24 +19,49 @@ def train_loop(dataset, model, optimizer):
         print(loss)
 
 
-def fit_loop(dataset, model, optimizer):
-    dataset = dataset.shuffle(12)
-    dataset = dataset.repeat(20)
-    dataset = dataset.batch(4)
+def fit_loop(dataset, model, optimizer, nepoch, batchsize):
+    nstep = len(list(dataset)) // batchsize
+    dataset = dataset.shuffle(12)  # 12 = buffer size
+    dataset = dataset.repeat(nepoch)
+    dataset = dataset.batch(batchsize)
     model.compile(optimizer=optimizer,
                   loss=tf.keras.losses.binary_crossentropy,
                   metrics=["accuracy"])
-    model.fit(dataset, epochs=20, steps_per_epoch=2)
+    model.fit(dataset, epochs=nepoch, steps_per_epoch=nstep)
+
+
+def get_args(args):
+    argparser = argparse.ArgumentParser()
+    argparser.add_argument('--imagepath', default='../../rainforest/fixed-train-jpg/',
+                           help='path to image directory')
+    argparser.add_argument('--labelpath', default='../../rainforest/train_v3.csv',
```

Comment:
Even if technically correct, it is common to set the style to everything on one line of one line per hyperparameters and no arbitrary break.

`argparser.add_argument('--imagepath', default='../../rainforest/fixed-train-jpg/', help='path to image directory')`

or

```
    argparser.add_argument('--seed', 
                                             default=-1, 
                                             type=int,
                                             help='Set random seed to this number (if >0)')
```

Reply:
ok, solved